packages/payment-integration-api/src/payment-integration-selectors.ts

```diff
@@ -49,7 +49,7 @@ export default interface PaymentIntegrationSelectors {
     getPaymentRedirectUrlOrThrow(): string;
 
     getPaymentMethod(methodId: string, gatewayId?: string): PaymentMethod | undefined;
-    getPaymentMethodOrThrow(methodId: string, gatewayId?: string): PaymentMethod;
+    getPaymentMethodOrThrow<T = any>(methodId: string, gatewayId?: string): PaymentMethod<T>;
```

Comment:
Can I know why adding `<T = any>` here?

Reply:
Because this method returns instance of `PaymentMethod` and this interface has generic type for `initializationData`
[https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/payment-integration-api/src/payment/payment-method.ts](https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/payment-integration-api/src/payment/payment-method.ts)

And if we need to [describe correct type](https://github.com/bigcommerce/checkout-sdk-js/pull/1808/files#diff-152a1fd0ab36037c134c9d74cdfdadc7780a8b2f5b053a95a0ebf3f0d11dfcfcR68) for `PaymentMethod.initializationData` we need to describe it as generic
any by default because it is using by default in [original](https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/payment-integration-api/src/payment/payment-method.ts) `PaymentMethod` interface

Reply:
Thanks @PavlenkoM can you try `T = unknown` instead of `any` here.

Reply:
Yes, you are right, with `unknown` looks better

Reply:
changing `any` to `unknown` effects other providers
lets leave `any` and change to `unknown` in separate PR?
<img width="1334" alt="Screenshot 2023-03-15 at 16 31 34" src="https://user-images.githubusercontent.com/9430298/225342094-770dc411-2fd4-4f96-b693-43e707e90335.png">

Reply:
PR with generic type fix
[https://github.com/bigcommerce/checkout-sdk-js/pull/1907](https://github.com/bigcommerce/checkout-sdk-js/pull/1907)